android/build.gradle

```diff
@@ -41,8 +41,8 @@ android {
 }
 
 repositories {
-  mavenCentral()
   google()
+  mavenCentral()
```

Comment:
What is the actual effect of changing this order?

Reply:
Just a bit better speed when building